api/views.py

```diff
@@ -2512,3 +2513,129 @@ def delete_research_in_price(request):
     current_coast_research.delete()
     Log.log(data["pk"], 130001, request.user.doctorprofile, data["data_json"])
     return JsonResponse({"ok": "ok"})
+
+
+@login_required
+@group_required('Конструктор: Настройка организации')
+def get_company_list(request):
+    company_data = [
+        {
+            "id": company.pk,
+            "title": company.title,
+        }
+        for company in Company.objects.filter(active_status=True)
+    ]
+    company_data = sorted(company_data, key=lambda company: company["title"])
+    return JsonResponse({"data": company_data})
+
+
+@login_required
+@group_required('Конструктор: Настройка организации')
+def get_contract_list(request):
+    contract_data = [
+        {
+            "id": contract.pk,
+            "label": contract.title,
+        }
+        for contract in Contract.objects.filter(active_status=True)
+    ]
+    contract_data = sorted(contract_data, key=lambda contract: contract["label"])
+    return JsonResponse({"data": contract_data})
+
+
+@login_required
+@group_required('Конструктор: Настройка организации')
+def get_current_company(request):
+    request_data = json.loads(request.body)
+    current_company = Company.objects.get(pk=request_data["id"])
+    company_data = {
+        "short_title": current_company.short_title,
+        "legal_address": current_company.legal_address,
+        "fact_address": current_company.fact_address,
+        "inn": current_company.inn,
+        "ogrn": current_company.ogrn,
+        "kpp": current_company.kpp,
+        "bik": current_company.bik,
+        "contract_id": current_company.contract.pk,
```

Comment:
Лучше будет
```suggestion
        "contract_id": current_company.contract_id,
```

Иначе:
1. Это дополнительный запрос в базу
2. Если contract пуст, то должна быть ошибка

Reply:
2 - важно, потому что поле contract в компании null=True